### DISCUSSION TITLE
Windows 11 compatibile

### DISCUSSION THREAD

**Discussion by ghost** [jamulussoftware/jamulus#3200] [Technical Support & Help] [answered]
Hello,

I'm wondering whether Jamulus is windows 11 compatibile? I work in an enterprise that will soon be updating to windows 11.

**Comment by ann0see** [comment, marked as answer]
Yes, it should work fine on windows 11. I currently don't have Win 11, but in the past I tested it and it worked.

**Reply by ghost** [reply]
Cool. Thanks. Don't currently have a copy of 11 yet so I haven't managed to test it myself.

**Reply by mcfnord** [reply]
Using it daily on Windows 11 with no problems.
